docs/cert-manager.md

```diff
@@ -11,67 +39,187 @@ by [cert-manager](https://cert-manager.io/).
 
 ```yaml
+# tenant-1-minio-certificate.yaml
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: tenant-certmanager-cert
-  namespace: minio-tenant
+  namespace: tenant-1
 spec:
   dnsNames:
-    - "*.tenant-certmanager.svc.cluster.local"
-    - "*.myminio.tenant-certmanager.svc.cluster.local"
-    - "*.myminio-hl.tenant-certmanager.svc.cluster.local"
+    - "minio.tenant-1"
+    - "minio.tenant-1.svc"
+    - '*.minio-tenant-1.svc.cluster.local'
+    - '*.minio.tenant-1.svc.cluster.local'
+    - '*.myminio-hl.tenant-1.svc.cluster.local'
   secretName: tenant-certmanager-tls
   issuerRef:
     name: tenant-certmanager-issuer
 ```
 
-Then it creates a new tenant including the new `tenant-certmanager-tls` secret in the `externalCertSecret` field.
+```shell
+kubectl apply -f tenant-1-minio-certificate.yaml
+```
+
+### Setup tenant to use certificate created by cert-manager 
+
+In the tenant spec make sure to do the following steps:
+
+* Disable `spec.requestAutoCert` as this will instruct Operator to do not attempt to issue certificates and instead rely on Cert Manager to provide them on secrets.
+* Reference the Secret containing the TLS certificate we just issued in the previous step in `spec.externalCertSecret`.
+
+Here is an example of Tenant sections should look like:
 
 ```yaml
+apiVersion: minio.min.io/v2
+kind: Tenant
+metadata:
+  name: myminio
+  namespace: tenant-1
+spec:
+  ## Disable default tls certificates.
+  requestAutoCert: false
   ## Use certificates generated by cert-manager.
   externalCertSecret:
     - name: tenant-certmanager-tls
       type: cert-manager.io/v1
 ```
 
-### Create `operator-ca-tls` secret
+## Trust CA in MinIO Operator
+
+MinIO Operator can trust as much CA certificates as provided, just create a secret with the prefix `operator-ca-tls-` 
+followed by an identifier in the `minio-operator` namespace, in this example the secret name will be `operator-ca-tls-tenant-1`.
+
+MinIO Operator is going to mount and trust all certificates issued by the provided CA's, this is specially needed because
+Operator performs health checks from MinIO in the health cluster endpoint (see more in docs https://min.io/docs/minio/kubernetes/upstream/operations/monitoring/healthcheck-probe.html#cluster-write-quorum).
 
-Copy the cert-manager CA from the tenant certificate, this will allow Operator to trust the cert-manager CA and allow Operator to trust the Tenant certificate
+Missing to instruct Operator to trust the MinIO Certificate (or it's CA) will cause an error log message in the Operator Pod logs as following:
 
+```error
+Failed to get cluster health: Get "https://minio.tenant-1.svc.cluster.local/minio/health/cluster":
+x509: certificate signed by unknown authority
+```
+
+### Create `operator-ca-tls-tenant-1` secret
+
+Copy the cert-manager generated CA certificate into the minio-operator namespace, this will allow Operator to trust the cert-manager issued CA and it's derivatives.
```

Comment:
```suggestion
Copy the cert-manager generated CA certificate into the minio-operator namespace, this will allow Operator to trust the cert-manager issued CA and its derivatives.
```